pkg/time/tickerwithtimeout.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2023 The Okteto Authors
```

Comment:
should we name this `ticker_timeout.go` to make it easier to read?

Reply:
There are some mixed opinions on file naming, some say it must be only lower case according to some golang conventions. Particularly, I find underscores useful

Reply:
I removed the whole pkg so we don't have this problem anymore 😜

Reply:
Thanks for suggesting that @AgustinRamiroDiaz, I've added an item (cc'd you on Notion). Happy to see feedback!